test/test_reporter.rb

```diff
@@ -30,5 +31,6 @@ def test_flakey_suite
     assert_match "Flaky jobs detected", output
     assert_match "./spec/foo_spec.rb:2", output
     refute_match "Failed examples", output
+    assert_match "bin/rspec --seed", output
```

Comment:
We should really assert the full command here (e.g. `bin/rspec --seed 123 .spec/foo_spec.rb:2`), otherwise we're not testing that the seed reported is the proper one. So we want to `exec_build` with a fixed seed, and verify that that seed is included in the reporter's output.

Ditto for `#test_failing_suite`.

Reply:
This comment belongs to [another PR](https://github.com/skroutz/rspecq/pull/54) but i will consider it.